modules/nw-sriov-network-attachment.adoc

```diff
@@ -38,45 +54,49 @@ metadata:
   namespace: openshift-sriov-network-operator <2>
 spec:
   networkNamespace: <target_namespace> <3>
-  ipam: <ipam> <4>
+  resourceName: <sriov_resource_name> <4>
   vlan: <vlan> <5>
-  resourceName: <sriov_resource_name> <6>
-  linkState: <link_state> <7>
-  maxTxRate: <max_tx_rate> <8>
-  minTxRate: <min_rx_rate> <9>
-  vlanQoS: <vlan_qos> <10>
-  spoofChk: "<spoof_check>" <11>
+  spoofChk: "<spoof_check>" <6>
+ifdef::ocp-sriov-net[]
+  ipam: <ipam> <7>
+  linkState: <link_state> <8>
+  maxTxRate: <max_tx_rate> <9>
+  minTxRate: <min_rx_rate> <10>
+  vlanQoS: <vlan_qos> <11>
   trust: "<trust_vf>" <12>
   capabilities: <capabilities> <13>
+endif::ocp-sriov-net[]
 ----
 <1> Replace `<name>` with a name for the CR. The SR-IOV Network Operator creates a NetworkAttachmentDefinition CR with same name.
 <2> Specify the namespace where the SR-IOV Operator is installed.
 <3> Optional: Replace `<target_namespace>` with the namespace where the NetworkAttachmentDefinition CR is created. The default value is `openshift-sriov-network-operator`.
-<4> Optional: Replace `<ipam>` a configuration object for the ipam CNI plug-in as a YAML block scalar. The plug-in manages IP address assignment for the attachment definition.
+<4> Replace `<sriov_resource_name>` with the value for the `.spec.resourceName` parameter from the SriovNetworkNodePolicy CR that defines the SR-IOV hardware for this additional network.
```

Comment:
Please specify if it's mandatory or optional.
Also if if namespace  provided here doesn't exist, how it should behave.

Reply:
Docs convention is that we mark up what is strictly 'optional'. All other options are presumed mandatory or best practice.
Also is this comment for L73? It refers to the resource name of the SriovNetworkNodePolicy. Not sure how namespace comes into it. Perhaps I misunderstand

Reply:
@aburdenthehand that means resourceName is mandatory to have. Thanks